source/administration/changelog.md

```diff
@@ -4,11 +4,233 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+- Using a mouse to select a channel in the channel switcher (CTRL/CMD+K) now switches to the correct channel.
+
+#### Markdown Text Formatting
+- Added a margin for Markdown inline images.
+- Improved Markdown heading sizes in the desktop view.
+
+#### On-Boarding
+- Added "Already have an account? Click here to sign in" link to the sign up page.
+- Improved experience of joining a team using an invite link.
+
+#### Files
+- SVG files now render in file preview.
+
+#### CLI Tool
+- Added new CLI commands:
+    - `platform check` for validating the `config.json` file.
+    - `platform search` for searching users based on username, email or user ID.
+
+#### OAuth 2.0 Service Provider
+- OAuth 2.0 service provider now always returns the refresh token.
+- New refresh token now issued when granting a new access token.
+
+#### System Console
+ - Added a confirmation dialog when deactivating a user.
+ - Server logs are now always printed in English regardless of Default Server Language, for easier troubelshooting.
+
+#### Enterprise Edition
+ - Added new [performance monitoring metrics](https://docs.mattermost.com/deployment/metrics.html) for
+     - The total number of connections to all the search replica databases
+     - The total number of WebSocket broadcasts sent
+
+### Bug Fixes
+- Long custom emoji names no longer float out of the emoji picker.
+- Deleted custom emoji no longer stay in "recently used" section of the emoji picker.
+- The maximum length of field "Position" increased to 64 characters. The previous limit caused problems with LDAP synchronization.
+- Pinning a post in center channel no longer changes pinned posts list in the right-hand sidebar.
+- Pinning a post in center now adds the badge to search results.
+- Fixed error message text for **Edit URL** field in channel creation modal.
+- Disabled config file watcher while running from makefile.
+- Fixed Go client's `GetTeamByName()` function.
+- Recent mentions search now properly includes `@[username]` in the search.
+- Updated error message when entering a password longer than maximum number of characters.
+- Don't send the same message multiple times when hitting "Retry" on a failed post.
+- Deleting your own pinned post in a direct message channel now properly disappear from the pinned posts list.
+- Fixed channel purpose help text for private channels.
+- When ability to change the header is restricted, "Set a Header" option is no longer shown in the channel intro.
+- Mention notifications now trigger if the word is formatted in bold, italic or strikethrough.
+- Mention notifications now longer trigger if the word is inside a code block.
+- In mobile view, Manage Members menu option no longer reads "View Members" for channel admins.
+- Fixed vertical space between posts in the right-hand sidebar.
+- Fixed flagged posts and recent mentions icon sizing in the header.
+- Fixed vertical space below link previews.
```

Comment:
The spacing stuff seems picayune... is it worth mentioning? Were these fixed due to customer request?